src/ORM/Rule/IsUnique.php

```diff
@@ -67,19 +67,20 @@ public function __invoke(EntityInterface $entity, array $options)
         if (!$entity->extract($this->_fields, true)) {
             return true;
         }
+        $this->_options = $options + $this->_options;
```

Comment:
I'm not sure we should be mutating state on each call. Could this be `$options = $options + $this->_options`?